docs/reference/modules/network.asciidoc

```diff
@@ -35,11 +34,23 @@ in the following table:
 
 |`_local:ipv6_` |Will be resolved to loopback IPv6 addresses
 
-|`_non_loopback_` |Addresses of the first non loopback interface
+|`_site_` |Will be resolved to site-local addresses ("private network")
```

Comment:
That's fine. I hadn't heard the term before and wikipedia has it as a redirect to "Unique local address" which looks like its just an IPv6 thing. And it has a link to an RFC that deprecates site local addresses. I guess my point is that the whole thing is muddied by years of standards bodies.

I still like the name `_site_` because its way more obvious than `_private_` or `_unique_` would be.